examples/E2E-RAG-App/README.md

```diff
@@ -0,0 +1,63 @@
+## E2E-RAG-App
+
+This is an End to End RAG App leveraging llama-stack that handles the logic for ingesting documents, storing them in a vector db and providing an inference interface.
+
+We share the details of how to run first and then an outline of how it works:
+
+### Prerequisite:
+
+Install docker: Check [this doc for Mac](https://docs.docker.com/desktop/setup/install/mac-install/), [this doc for Windows](https://docs.docker.com/desktop/setup/install/windows-install/) and this [instruction for Linux](https://docs.docker.com/engine/install/).
+
+For Mac and Windows users, you need to start the Docker app manually after installation.
+
+### How to run the pipeline:
+
+![RAG_workflow](./RAG_workflow.jpg)
```

Comment:
any chance we can simplify this diagram a _lot_? actually, I think a simpler inline Mermaid diagram which shows just the basic high-level flow would be more useful. docker, etc. should be completely avoided.

Reply:
Sure.. I will make a short version